### ISSUE TITLE
Contao 5.3 - diverse Fehlermeldungen beim Filterformular Inhaltselement

### ISSUE THREAD

**Issue by kreadiv** [alnv/catalog-manager#394]
`
    public function editCatalogForm(DataContainer $dc)
    {
        return ($dc->value < 1) ? '' : ' <a href="contao/main.php?do=filterform&amp;table=tl_catalog_form_fields&amp;id=' . $dc->value . '&amp;popup=1&amp;nb=1&amp;rt=' . REQUEST_TOKEN . '" title="' . sprintf(specialchars($GLOBALS['TL_LANG']['tl_content']['editalias'][1]), $dc->value) . '" style="padding-left:3px" onclick="Backend.openModalIframe({\'width\':768,\'title\':\'' . specialchars(str_replace("'", "\\'", sprintf($GLOBALS['TL_LANG']['tl_content']['editalias'][1], $dc->value))) . '\',\'url\':this.href});return false">' . \Image::getHtml('alias.gif', $GLOBALS['TL_LANG']['tl_content']['editalias'][0], 'style="vertical-align:top"') . '</a>';
    }
`

- REQUEST_TOKEN gibt es nicht mehr in Contao 5.3
- specialchars geht auch nicht. Habe ich ersetzt durch htmlspecialchars.
- Image kann nicht mehr aus dem globalen Namensraum aufgerufen werden. Habe ich per use Statement importiert.

**Comment by alnv** [comment]
Diese Fehler betreffen die v2.0, oder?

**Comment by kreadiv** [comment]
Genau, die v2.0, welche auf https://extensions.contao.org/ aktuell unter "Catalog Manager" angezeigt wird. Lese aber gerade, dass die v2.0 deprecated ist und stattdessen die Version 3.0 verwendet werden soll. Damit hat sich die Frage erledigt. Danke!
